src/ImageSharp/Formats/Tiff/TiffFrameMetadata.cs

```diff
@@ -0,0 +1,289 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Linq;
+using SixLabors.ImageSharp.Common.Helpers;
+using SixLabors.ImageSharp.Formats.Tiff.Compression;
+using SixLabors.ImageSharp.Formats.Tiff.Constants;
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.Metadata.Profiles.Exif;
+
+namespace SixLabors.ImageSharp.Formats.Tiff
+{
+    /// <summary>
+    /// Provides Tiff specific metadata information for the frame.
+    /// </summary>
+    internal class TiffFrameMetadata : IDeepCloneable
```

Comment:
This class really isn't doing much as it is, really acting as a proxy during decode and I don't think it should be purposed as such since it's functionality designed for different purposes. 

E.G. In the future I see the type containing non-exif things like jpeg quality or bits per pixel information.

I would suggest moving the property extraction methods to a static method for verification and working directly against the profile entries for now during decode.

Reply:
I have removed almost all properties from `TiffFrameMetaData` and do access them now via the profile directly, see 8461c72e

The only Properties left are `BitsPerPixel` and `BitsPerSample`